src/Pi.cpp

```diff
@@ -912,6 +927,19 @@ void Pi::App::PostUpdate()
 	GuiApplication::PostUpdate();
 
 	HandleRequests();
+
+#ifdef PIONEER_PROFILER
+	// TODO: profileSlow is profiling the previous frame, need to move that functionality to Application
+	if (Pi::doProfileOne || (Pi::doProfileSlow && (GetFrameTime() > 0.1))) { // slow: < ~10fps
+		Pi::doProfileOne = false;
+		if (!Pi::profileOnePath.empty()) {
+			Profiler::dumphtml(Pi::profileOnePath.c_str());
+			Pi::profileOnePath.clear();
+		} else {
+			Profiler::dumphtml(Pi::profilerPath.c_str());
+		}
+	}
+#endif
```

Comment:
Are the profiling-related patches meant to go in within this PR or is it just to make the development easier but will go away in the non-draft version?

If the former, would it be possible to move them in a separate PR if that's not too much work ? At a glance the changes don't seem coupled with the equipment/economy stuff, but I might be wrong.

Reply:
Profiling will probably be committed directly to master, yeah. It was work that was kicking around in the branch and I didn't want to separate it out yesterday. I don't think this is particularly large enough for a separate PR (especially given that we just merged in a bunch of profiler changes), given that the new method is only really used thrice.